AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -124,35 +125,46 @@ def AtomEditorComponents_Light_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.creation_redo, light_entity.exists())
 
-        # 5. Enter/Exit game mode.
+        # 5. Cycle through all light types.
+        for ltype, value in LIGHT_TYPES.items():
+            light_component.set_component_property_value(
+                AtomComponentProperties.light('Light type'), LIGHT_TYPES[ltype])
```

Comment:
this will set 'unknown' value 0 to the component property. is that valid?

Reply:
I assumed it was valid because it shows up in the LIGHT_TYPE list

if you disagree, remove it from the LIGHT_TYPE list

Reply:
it's in LIGHT_TYPE but it's essentially the unset value for the type as in you haven't selected a light type yet. It's probably fine to use it but a light would not have a type with this set so further testing with it set would be limited.
we shouldn't remove it from LIGHT_TYPE dictionary